lintcheck/src/input.rs

```diff
@@ -10,6 +10,10 @@ use walkdir::{DirEntry, WalkDir};
 
 use crate::{Crate, LINTCHECK_DOWNLOADS, LINTCHECK_SOURCES};
 
+const DEFAUTL_DOCS_LINK: &str = "https://docs.rs/{krate}/{version}/src/{krate}/{file}.html#{line}";
+const DEFAUTL_GITHUB_LINK: &str = "{url}/blob/{hash}/src/{file}#L{line}";
+const DEFAUTL_PATH_LINK: &str = "{path}/src/{file}:{line}";
```

Comment:
Ah one last thing - typo in `DEFAULT`, missed that until now 😄

Reply:
*That was totally intentional, and just a check if you actually check the changes /s :P*